packages/preview/charged-vde/1.0.0/template/main.typ

```diff
@@ -0,0 +1,71 @@
+#import "@preview/charged-vde:1.0.0": charged-vde
+//#import "../lib.typ": vde
```

Comment:
Can you remove this line, please? It doesn't make sense for users of the template, that won't have a copy of the whole package.